src/modules/users/users.repository.ts

```diff
@@ -16,19 +20,72 @@ export class UsersRepository {
         email,
         password,
         displayName: `${name} ${lastName}`,
+        refreshToken: '',
```

Comment:
Instead of harcode a specific refresh token, I should change this attribute to be null at db level.